### PR TITLE
New bad years for Djibouti; systematize one-off data conversion scripts

### PR DESCRIPTION
**Pull request by aaron-kaplan**
I have copied all the source data files, including the one from /data/kgraaf/original-data, into /data/aaron/fbf/original-data.

### CODE REVIEW

File: fbfmaproom/data-conversion-scripts/niger/station-spi.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import xarray as xr
+import cftime
+
+fname = "/data/aaron/fbf/original-data/niger/Cumul_juine_juiy_monthly_rainfall_1991_2022.xlsx"
+data = pd.read_excel(fname, sheet_name="array and analysis", usecols="AQ", skiprows=42, nrows=32)
+data_fixed = pd.read_excel(fname, sheet_name="array and analysis", usecols="AS", skiprows=42, nrows=32)
+data['Average'][data.index[-1]] = data_fixed['Unnamed: 44'][data_fixed.index[-1]]
+t = pd.read_excel(fname, sheet_name="array and analysis", usecols="AR", skiprows=42, nrows=32)
+T = t.apply(lambda y: cftime.datetime(y, 8, 16, calendar='360_day'), axis=1)
+xr.Dataset(data.rename(T)).rename_dims({'dim_0': 'T'}).rename_vars({'dim_0': 'T'}).to_zarr("/data/<<DATADIR>>/niger/station-spi-jj.zarr")
```
Comment (kgraaf) [inline fbfmaproom/data-conversion-scripts/niger/station-spi.py:14]: change to relative directory so this is in theory runnable

**Review by kgraaf — CHANGES_REQUESTED** [review]

File: fbfmaproom/data-conversion-scripts/niger/station-spi.py (hunk 1 above)
Comment (aaron-kaplan) [reply fbfmaproom/data-conversion-scripts/niger/station-spi.py]: done.

**Review by kgraaf — APPROVED** [review]

**Comment by aaron-kaplan** [comment]
@kgraaf This shouldn't have been merged, because FIST hasn't approved it yet. In general I think it should be the author who merges, in the absence of any particularly urgency, because the author usually knows the context best.

**Comment by aaron-kaplan** [comment]
I just pinged FIST again. If they don't reply in the next couple hours, I'll send a PR to revert this. Don't want to risk it going to prod prematurely.

**Comment by kgraaf** [comment]
Sorry about that, I totally forgot!

**Comment by aaron-kaplan** [comment]
Actually none of the deployed code is changed in this PR, so even if we release this version it won't change anything. We do need to remember not to copy the config file changes to dlconfig or sync the dataset to /data/aaron/fbf until it's approved.

**Pull request merged**